gc/verbose/VerboseHandlerOutput.cpp

```diff
@@ -482,6 +502,14 @@ MM_VerboseHandlerOutput::handleCycleEnd(J9HookInterface** hook, uintptr_t eventN
 	} else {
 		writer->formatAndOutput(env, 0, "<cycle-end %s />", tagTemplate);
 	}
+
+	uint64_t fixupDuration = event->fixHeapForWalkTime;
+
+	if (fixupDuration > 0) {
```

Comment:
slightly more cleaner if you base it on fixHeapForWalkReason being different than NONE